### ISSUE TITLE
SSH key issue

### ISSUE THREAD

**Issue by deeppatelv** [bomboradata/datascience-exam#98]
As I am working on my college laptop there was an issue with the SSH key so It was not allowing me to clone. So I created my own repository and worked on that and made changes in that only.

**Comment by NahsiN** [comment]
Yes you don't have permissions to clone this repo. You can only _fork_ it which is exactly what you did. I am going to update the README to make this more clear. Thanks
